packages/api/src/auth/github/github.service.ts

```diff
@@ -9,7 +9,7 @@ import { UserFromGithub } from '../../users/users.types';
 export class GithubService {
   constructor(private readonly usersRepository: UsersRepository) {}
 
-  async authorizeUser(githubUserData: UserFromGithub): Promise<User> {
+  async authorizeUser(githubUserData: UserFromGithub): Promise<User | RegisteredUser> {
```

Comment:
I don't know why here is returned User OR RegisteredUser? All those property all important? Maybe just id or something like that can be returned?